assets/js/app.js

```diff
@@ -21,14 +21,9 @@ import "phoenix_html";
 import { Socket } from "phoenix";
 import { LiveSocket } from "phoenix_live_view";
 import topbar from "../vendor/topbar";
-import ImageCropper from "./hooks/cropper";
 
 let Hooks = {};
```

Comment:
Is that the cleaned up version? I don't know a lot about JavaScript. I just had a look at https://github.com/animina-dating/animina/blob/1076d78d3eb7611ec7b75ef45823127bd940d358/assets/js/app.js and I want to be sure that we have a clean sweep here.

Reply:
I removed any reference to the ImageCropper hook